federation/apis/federation/install/install_test.go

```diff
@@ -68,7 +68,7 @@ func TestCodec(t *testing.T) {
 
 func TestInterfacesFor(t *testing.T) {
 	if _, err := api.Registry.GroupOrDie(federation.GroupName).InterfacesFor(federation.SchemeGroupVersion); err == nil {
-		t.Fatalf("unexpected non-error: %v", err)
+		t.Fatalf("expected an error,but got nil")
```

Comment:
please add a space after the comma. 
and amend the commit message to be a more meaningful one.